cmd/kpod/rm.go

```diff
@@ -39,20 +44,43 @@ func rmCmd(c *cli.Context) error {
 	defer runtime.Shutdown(false)
 
 	args := c.Args()
-	if len(args) == 0 {
+	if len(args) == 0 && !c.Bool("all") {
 		return errors.Errorf("specify one or more containers to remove")
 	}
 
-	for _, container := range args {
+	var delContainers []string
```

Comment:
Can avoid duplicate lookups if you make this a []*Container and set it to runtime.GetContainers() if --all is set, and do LookupContainer on args otherwise. Makes the for loop a little simpler, though it does add another loop.

Reply:
This would make dealing with lastError a lot harder than it is now, though. I guess I'm undecided as to whether this is a good idea.

Reply:
this makes error handling for bad input a little wonky, but I agree with the suggestion ... fixed.